test-run.py

```diff
@@ -91,6 +91,8 @@ def main_loop_parallel():
 
     color_stdout("Timeout options:\n", schema='tr_text')
     color_stdout('-' * 19, "\n",       schema='separator')
+    color_stdout("SERVER_START_TIMEOUT:" . ljust(26) + "{}\n" .
```

Comment:
Test for a new timeout is missed. Could you add it?

Reply:
Added unit test is about this timeout as well.